packages/contracts/deploy/verify.ts

```diff
@@ -3,71 +3,152 @@ import {DeployFunction} from 'hardhat-deploy/types';
 import {TASK_ETHERSCAN_VERIFY} from 'hardhat-deploy';
 
 import {verifyContract} from '../utils/etherscan';
+import {getContractAddress} from './helpers';
 
 function delay(ms: number) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log('\nVerifying contracts');
+
   const {deployments, ethers, run} = hre;
 
-  console.log('Verifying registry and factories contracts');
+  const msDelay = 2; // 3 minutes - Etherscan needs some time to process before trying to verify.
```

Comment:
rename to only `delay` or `minutesDelay`. the value in this constant isn't in ms

Reply:
amended